### PR TITLE
bug 1590599 - wrap winsign.sign.sign_file calls in retry_async

### PR DESCRIPTION
**Pull request by escapewindow**
I think we contact timestamp.digicert.com in this function, so we need
retries to be able to handle network/service blips.

### CODE REVIEW

**Comment by escapewindow** [comment]
Related: https://mozilla-hub.atlassian.net/browse/SVCSE-484

**Review by jcristau — APPROVED** [review]
lgtm

File: signingscript/src/signingscript/sign.py [hunk 1]
```diff
@@ -1328,20 +1334,19 @@ async def signer(digest, digest_algo):
...
-        cafile=cafile,
-        timestampfile=timestampfile,
-        url=url,
-        comment=authenticode_comment,
-        crosscert=crosscert,
-        timestamp_style=timestamp_style,
-    ):
-        raise IOError(f"Couldn't sign {orig_path}")
+    # Retry winsign.sign.sign_file, because the timestamp server can hiccup
+    await retry_async(
```
Comment (gbrownmozilla) [inline signingscript/src/signingscript/sign.py:1338]: You are happy with default args for retry_async attempts, sleeptime, etc? (Just checking - not suggesting a change.)

**Review by gbrownmozilla — APPROVED** [review]

File: signingscript/src/signingscript/sign.py (hunk 1 above)
Comment (escapewindow) [reply signingscript/src/signingscript/sign.py:1338]: Yeah, I'm not sure which retry args to specify, and I generally default to the default unless I have an opinion.

**Pull request merged**